weblab/static/js/db.js

```diff
@@ -20,7 +20,8 @@ function submitNewExperiment(jsonObject, $td, entry)
 
   $td.append("<img src='"+staticPath+"img/loading2-new.gif' alt='loading' />");
 
-  $.post('/experiments/new', jsonObject, function(data) {
+//  $.post('/experiments/new', jsonObject, function(data) {
+  $.post($("#matrixdiv").data('new-href'), jsonObject, function(data) {
```

Comment:
```suggestion
  $.post($("#matrixdiv").data('new-href'), jsonObject, function(data) {
```

Similarly elsewhere - don't keep commented out lines :)